docs/tutorials/nav2-integration.md

```diff
@@ -1,103 +1,83 @@
 
-:::{figure} ../_images/beluga_amcl.png
-:alt: beluga_amcl component in beluga_nav2_launch.py from beluga_demo_nav2 package.
+:::{figure} ../_images/nav2_to_beluga_code_migration.gif
+:alt: Using the `beluga_amcl` component instead of `nav2_amcl`'s.
 :align: center
+:scale: 150%
 
-Loading `beluga_amcl` composable node, provided by the `beluga_demo_nav2` package.
+Loading `beluga_amcl` composable node, instead of `nav2_amcl`'s.
 :::
 
-:::{figure} ../_images/nav2_amcl.png
-:alt: nav2_amcl component in localization_launch.py from nav2_bringup package.
-:align: center
+That is what the `beluga_demo_nav2_integration` package does. Even the same parameter file can be used without changes, though the `beluga_demo_nav2_integration` does apply a few minimal changes to better configure `beluga_amcl`, as explained in its reference page.
 
-Loading of `nav2_amcl` composable node, provided by the `nav2_bringup` package.
-:::
+## Run the code
+
+To keep things as simple as possible, the `beluga-demos` repository comes with a containerized environment for demo execution:
+
+```bash
+cd beluga-demos
+./docker/run.sh
+```
 
-That is what the `beluga_demo_nav2` package does. Even the same parameter file can be used unchanged, though `beluga_demo_nav2` does apply a few minimal changes to better configure `beluga_amcl`, as explained in its reference page.
+Once inside, you can build all demos with:
 
-## Run the example
+```bash
+demo_build
+```
 
-Build the workspace:
+You can also make do without containers. You will have to install [ROS 2 Humble](https://docs.ros.org/en/humble/Installation/Ubuntu-Install-Debians.html) and `rosdep install` demo dependencies before you can `colcon build` it:
 
 ```bash
 source /opt/ros/humble/setup.bash
 cd ~/beluga_demo_ws
-colcon build --symlink-install --packages-up-to beluga_demo_nav2
+rosdep install -i -y --from-path src
+colcon build --symlink-install --packages-up-to beluga_demo_nav2_integration
 ```
 
-Launch the system:
+After building, you can launch the demo with:
 
 ```bash
-source install/local_setup.bash
-ros2 launch beluga_demo_nav2 bringup_launch.py
+source install/setup.bash
+ros2 launch beluga_demo_nav2_integration demo_office_navigation.launch.py
 ```
 
-You should see `rviz2` open with a map loaded and the robot initialized in an incorrect position. You should also see Gazebo open with both TurtleBot 3 world and robot spawned.
+This is equivalent to the `nav2_integration_demo` command that is available within the `beluga-demos` container.
 
-:::{image} ../_images/beluga_nav2_rviz.png
-:alt: beluga_amcl component in beluga_nav2_launch.py from beluga_demo_nav2 package.
-:align: center
-:::
+You should see Gazebo open up an office scenario with a TurtleBot in it. You should also see `rviz2` open with the corresponding map loaded.
 
-:::{image} ../_images/beluga_nav2_gazebo.png
-:alt: beluga_amcl component in beluga_nav2_launch.py from beluga_demo_nav2 package.
+:::{image} ../_images/beluga_nav2_demo.gif
```

Comment:
We have cuter office environments now, but requires updating the gazebo version.

https://github.com/Ekumen-OS/andino_gz

Let's not do it in this MR, buy let's create a ticket for the future.